### ISSUE TITLE
driver.h not found in wdt

### ISSUE THREAD

**Issue by manageryzy** [kendryte/kendryte-freertos-demo#3]
```
[  3%] Built target fatfs
[  7%] Built target hal
[ 21%] Built target freertos
[ 45%] Built target bsp
[ 47%] Built target drivers
[ 48%] Building C object CMakeFiles/wdt.dir/mnt/c/tools/kendryte/kendryte/kendryte-freertos-demo/wdt/main.c.obj
/mnt/c/tools/kendryte/kendryte/kendryte-freertos-demo/wdt/main.c:18:10: fatal error: driver.h: No such file or directory
 #include <driver.h>
          ^~~~~~~~~~
compilation terminated.
CMakeFiles/wdt.dir/build.make:62: recipe for target 'CMakeFiles/wdt.dir/mnt/c/tools/kendryte/kendryte/kendryte-freertos-demo/wdt/main.c.obj' failed
make[2]: *** [CMakeFiles/wdt.dir/mnt/c/tools/kendryte/kendryte/kendryte-freertos-demo/wdt/main.c.obj] Error 1
CMakeFiles/Makefile2:71: recipe for target 'CMakeFiles/wdt.dir/all' failed
make[1]: *** [CMakeFiles/wdt.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by manageryzy** [comment]
@sunnycase why hpp

**Comment by vowstar** [comment]
May should update all demos?

**Comment by sunnycase** [comment]
Some demos are not up-to-date and they will be soon.

**Comment by sunnycase** [comment]
For workaround you can remove this line. `driver.h` is not a public header file and should not be included.

**Comment by sunnycase** [comment]
Fixed in dfc19cd2307df25c6282f1184508fadaa5a1d73f.
